server/chat-plugins/poll.ts

```diff
@@ -382,6 +388,7 @@ export const commands: ChatCommands = {
 				});
 				curRoom.sendMods(this.tr`(${user.name} deleted the queued poll in slot ${slot}.)`);
 				curRoom.update();
+				if (roomid) this.parse(`/j view-pollqueue-${curRoom}`);
```

Comment:
What is this line for?

Reply:
Oops it was for testing.
You know how to fix the 120 character limit problem ?